src/multisig/multisig_account_kex_impl.cpp

```diff
@@ -429,6 +434,7 @@ namespace multisig
     }
 
     // sanitize input messages
+    multisig_keyset_map_memsafe_t pubkey_origins_map;
```

Comment:
This seems unneeded.

Reply:
The definition of `pubkey_origins_map` in this function was removed, so this line adds it back in.